components/commons/blog/postPreviewItem/postPreviewItem.global.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="bg-light-surface dark:bg-dark-surface flex flex-col-reverse lg:flex-row mb-8 rounded py-4 px-6 hover:bg-gray-200 transition-colors duration-300 ease-linear">
```

Comment:
Card have hover state. But only click on link actually do anything. I suggest to wrap whole card in nuxt-link (you can use tag="article" like you did previously) so click anywhere on card will move user to next route.

Reply:
ok, but what about title wrapped into a tag ? thought was good practice for seo purpose

Reply:
Just leave it. But make wrapper clickable (but no nested `a` tags, this is invalid html).